license-bill-of-materials.go

```diff
@@ -488,7 +491,12 @@ func longestCommonPrefix(licenses []License) string {
 			break
 		}
 		for _, c := range n.Children {
-			prefix = append(prefix, c.Name)
+			if c.Shared > 1  {
```

Comment:
it skips if shared <= 1; /a/b/c/d/e and /a/b/c/d/e/f/g will have shared > 1 up to and including e. f will have shared == 1